webui/react/src/services/decoder.ts

```diff
@@ -632,19 +632,29 @@ export const decodeV1TrialToTrialItem = (data: Sdk.Trialv1Trial): types.TrialIte
   };
 };
 
+/**
+ * Group metrics get returned as a string version of float values.
+ * This is needed to bridge the gap in the mean time.
+ */
+const decodeMetricValues = (data: unknown): Record<string, number> => {
+  if (!isObject(data)) return {};
+  const dataObj = data as Record<string, unknown>;
+  return Object.keys(dataObj).reduce((acc, key) => {
+    acc[key] = isNumber(dataObj[key]) ? (dataObj[key] as number) : parseFloat(`${dataObj[key]}`);
```

Comment:
@hamidzr the `parseFloat` is a temporary measure for [this reported issue](https://hpe-aiatscale.atlassian.net/browse/DET-9736)

Reply:
Please document that in the code.